.github/ISSUE_TEMPLATE/02_bug.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: A concrete bug report with steps to reproduce the behavior.
+labels: bug,
+---
+
+### Bug description
+
+
+
+### How is the bug triggered?
+How can you reproduce the bug?
+1.
+
+
+### What did you expect to happen?
+
+
+
+### How are you running the application?
+Please include at least:
+- **Zulip-terminal version:**
+  eg. a specific version (0.7.0), or if running from `main` also ideally the git ref
+- **Zulip server version:**
+  eg. Zulip Cloud, or the version you are running self-hosted
+- **Operating system (and version):**
+  eg. Debian Linux, Ubuntu Linux, macOS, WSL in Windows
+
+If possible, please provide details from the `About` menu:
```

Comment:
Maybe we can also add the shortcut (Meta + ?) for the About Menu?

Reply:
Yeah, sounds reasonable - though we'll need to update this once we know how we're going to express `Meta` consistently.
```suggestion
If possible, please provide details from the `About` menu (<kbd>Meta</kbd>+<kbd>?</kbd>):
```